### ISSUE TITLE
The audio device is not set to idle when speaking is finished

### ISSUE THREAD

**Issue by btman16** [pumper42nickel/eloquence_threshold#24]
In other synth drivers for NVDA, including eSpeak-NG and SAPI5, the idle state is set when speech finishes, which allows audio ducking to work fully, including the "Duck when outputting speech and sounds" option. Setting this to idle also seems to provide longer battery life on laptops, as I have noticed that I get longer battery life when using other synth drivers as compared to this one.

Therefore I was wondering if it would be possible to support setting the audio device to idle when speech finishes please?

Thank you very much for your time and I look forward to any input you may have.

Sincerely,

Brandon Tyson

**Comment by pumper42nickel** [comment]
Right, I am aware that ducking doesn't work properly. Buffering logic that deals with that is quite complicated. @Mohamed00 - are you perhaps familiar with this part of code?
